bin/openaps-packages.sh

```diff
@@ -15,9 +15,19 @@ echo 'Acquire::ForceIPv4 "true";' | sudo tee /etc/apt/apt.conf.d/99force-ipv4
 apt-get install -y sudo
 sudo apt-get update && sudo apt-get -y upgrade
 sudo apt-get install -y git python python-dev software-properties-common python-numpy python-pip nodejs-legacy watchdog strace tcpdump screen acpid vim locate jq lm-sensors || die "Couldn't install packages"
-if ! sudo apt-get install -y npm; then
-    sudo bash -c "curl -sL https://deb.nodesource.com/setup_8.x | bash -" || die "Couldn't setup node 8"
-    sudo apt-get install -y nodejs || die "Couldn't install nodejs"
+#if ! sudo apt-get install -y npm; then
+# install/upgrade to node 8
+if ! nodejs --version | grep 'v8.'; then
+    if grep -qa "Explorer HAT" /proc/device-tree/hat/product &>/dev/null ; then
+        mkdir $HOME/src/node && cd $HOME/src/node
+        wget https://nodejs.org/dist/v8.10.0/node-v8.10.0-linux-armv6l.tar.xz
+        tar -xf node-v8.10.0-linux-armv6l.tar.xz
+        cd *6l
+        sudo cp -R * /usr/local/
```

Comment:
This could do something unintended if one of the above commands fail.  Probably want to chain some of them together with && or similar, or use absolute paths here.